### PR TITLE
Implement API for change log folder of partitions in TopicAdmin

### PR DESCRIPTION
**Pull request by qoo332001**
#75
在TopicAdmin實作搬移partitions的API以及在embedded cluster配置多個folders

### CODE REVIEW

**Review by chia7712 — COMMENTED** [review]
@qoo332001 Do you plan to update dockerfile in this PR? Or you can enhance it in another one.

**Comment by qoo332001** [comment]
>Do you plan to update dockerfile完成 in this PR? Or you can enhance it in another one.

I will try to update the dockerfile in this PR tomorrow, after this PR merged I'll create another PR to complete #88

File: app/src/main/java/org/astraea/topic/Builder.java [hunk 1]
```diff
@@ -50,6 +51,32 @@ public void close() {
                   .collect(Collectors.toSet()));
     }
 
+    @Override
+    public Set<String> brokerFolders(Integer broker) {
```
[the remark this answers is not in the document]
Comment (qoo332001) [reply app/src/main/java/org/astraea/topic/Builder.java:55]: Does the return need to be changed to ```Map<Integer,Set<String>>```?

[the remark this answers is not in the document]
Comment (chia7712) [reply app/src/main/java/org/astraea/topic/Builder.java:55]: > Does the return type need to be changed to Map<Integer,Set<String>>?

yep

File: app/src/test/java/org/astraea/service/Services.java [hunk 2]
```diff
@@ -65,7 +75,8 @@ public void close() {
               broker.shutdown();
               broker.awaitShutdown();
             });
-        tempFolders.forEach(Utils::delete);
+        for (var b : tempFolders.keySet())
```
Comment (chia7712) [inline app/src/test/java/org/astraea/service/Services.java:78]: `tempFolders.values().forEach(fs -> fs.forEach(f -> Utils.delete(new File(f))));`

**Comment by chia7712** [comment]
@qoo332001 整體改得很棒，麻煩把前面已經修改的comments點掉（resolve conversation)，另外修正一下標題，修好最後一個意見就可以合併了，謝謝

File: app/src/main/java/org/astraea/topic/TopicAdmin.java [hunk 3]
```diff
@@ -50,6 +50,12 @@ static TopicAdmin of(Map<String, Object> configs) {
   /** @return all brokers' ids */
   Set<Integer> brokerIds();
 
+  /**
+   * @param broker broker ID
+   * @return all log directory
+   */
+  Map<Integer, Set<String>> brokerFolders(Set<Integer> broker);
```
Comment (chia7712) [inline app/src/main/java/org/astraea/topic/TopicAdmin.java:57]: `broker` -> `brokers`

**Pull request merged**